docs/DeveloperGuide.md

```diff
@@ -97,7 +97,6 @@ This section includes the sources of code, documentation and third-party librari
    1. Navigate to `src/main/java/cooper/Cooper.java`
    2. Right click on `Cooper.java` and select 'Run Cooper.main()'.
    3. You should see the following output if the setup was done correctly:
-      
```

Comment:
Oh ya and also I think we need the line here also, I remember my ip UG nearly got screwed up cause i didn't add the extra empty line here